data-plane/scripts/amlight_kytos.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
+echo "192.168.0.6 mongo1t" >> /etc/hosts
+echo "192.168.0.7 mongo2t" >> /etc/hosts
+echo "192.168.0.8 mongo3t" >> /etc/hosts
 
```

Comment:
As I understood from PR#72, DNS service provided by docker-compose would manage these hostnames rather than /etc/hosts
https://github.com/atlanticwave-sdx/sdx-continuous-development/pull/72/files#diff-df23b20a8bcbe2c83c706fa5c6767a69d70fca331eb6b4e782be5b31885bf87e

Reply:
Good Morning Mert, correct I committed the change, Thank you